### PR TITLE
feat: added IP lists for nodes

### PR DESCRIPTION
**Pull request by tstaetter**

### CODE REVIEW

File: blockjoy/api/v1/node.proto [hunk 1]
```diff
@@ -92,6 +92,10 @@ message Node {
   // IP-address is stored in the `ip_address` field.
   optional string address = 12;
   string host_id = 13;
+  // Whitelisted IP address for firewall
+  repeated string allow_ips = 14;
```
Comment (loony-bean) [inline blockjoy/api/v1/node.proto:96]: not sure i follow, but don't we have ip's here? /cc @mjr-blockjoy 
https://github.com/blockjoy/api-proto/blob/main/blockjoy/api/v1/node.proto#L190

Comment (tstaetter) [reply blockjoy/api/v1/node.proto:96]: there's no IP type in protos

Comment (loony-bean) [reply blockjoy/api/v1/node.proto:96]: i mean we already pass ip addresses, in some different way

Comment (tstaetter) [reply blockjoy/api/v1/node.proto:96]: just as string, see line 83

**Review by loony-bean — APPROVED** [review]
lgtm

**Pull request merged**